### ISSUE TITLE
Updated title page design

### ISSUE THREAD

**Issue by Jassob** [ayberkt/chalmers-msc-thesis-template#28]
It seems that Chalmers has updated their title page design somewhat recently. At least according to [this example](https://student.portal.chalmers.se/sv/chalmersstudier/kandidat-och-examensarbete/examensarbete/Documents/Mall%20omslag%20till%20examensarbete%20Chalmers_med%20bild_EN.pdf) it seems quite different to the title page we use. 

Should we update the template to make it look like this? I am thinking mainly on the header and footer..

**Comment by ayberkt** [comment]
Yes, I saw this example but I think it is supposed to be just one instance of a thesis that conforms to the vague specification (that I linked to in the README). Maybe we should check this with Birgit?

**Comment by Jassob** [comment]
> [..] I think it is supposed to be just one instance of a thesis that conforms to the vague specification (that I linked to in the README). Maybe we should check this with Birgit?

Yes, I think we should. I don't know how much of the specification is guidelines and how much are rules..
